middleware/proxy/proxy.go

```diff
@@ -58,7 +58,9 @@ func Balancer(config Config) fiber.Handler {
 		cfg.Next = ConfigDefault.Next
 	}
 	if len(cfg.Servers) == 0 {
-		panic("Servers cannot be empty")
+		return func(c *fiber.Ctx) (err error) {
+			panic("Servers cannot be empty")
+		}
```

Comment:
Shouldn't we keep the code as it is and panic on MW creation?

Reply:
This has an issue where it cannot be caught by the `recover` middleware.